public/javascripts/chooseLanguage.js

```diff
@@ -0,0 +1,49 @@
+function chooseLanguage() {
+    function languageList(songPosition) {
+        return m('.col-xs-6', [
+            m('select.form-control', {
+                onchange: function() {
+                    if (songPosition === 'left') {
+                        window.location.href = window.location.href.replace(/left=../, 'left=' + $(this).val());
+                    } else {
+                        window.location.href = window.location.href.replace(/right=.*/, 'right=' + $(this).val());
+                    }
+                },
+                config: function(elem, isInit) {
+                    if (!isInit) {
+                        //getting the startIndex for langcode, +1 because of '='
```

Comment:
mention that the '=' is in the query string. I still don't get why we're reading from the query string this way...